src/web/js/events.js

```diff
@@ -86,11 +96,20 @@ function makeEvents(config) {
         const interactions = message.currentState.interactionsSinceLastRun;
         const src = interactions[interactions.length - 1];
         interactionsSinceLastRun.push(src);
-        const dontFocus = (e) => { e.preventDefault(); e.stopPropagation(); };
-        $(window).on("focusin", dontFocus);
+        const dontFocus = (e) => {
+          e.preventDefault();
+          e.stopPropagation();
+        };
```

Comment:
`dontFocus` isn't used anymore, can remove.

Reply:
Good catch, thanks Joe.

I removed this and the commented code